lib/hedwig_mopidy/responders/mopidy.ex

```diff
@@ -199,14 +208,40 @@ defmodule HedwigMopidy.Responders.Mopidy do
     send message, response
   end
 
-  hear ~r/^dj\svolume$/i, message do
+  hear ~r/^dj\s(skip|next)$/i, message do
+    currently_playing = HedwigMopidy.currently_playing
+    current_playlist = CurrentPlaylistStore.retrieve
+    response =
+      with {:ok, %TlTrack{} = next_track} <- Tracklist.next_track,
+           {:ok, :success} <- Playback.next do
+            HedwigMopidy.notice_message("Skipping to next track: #{HedwigMopidy.playing_string(next_track, current_playlist)}")
+      else
+        {:error, error_message} -> error_message
+        _ -> HedwigMopidy.notice_message("No more songs are queued")
+      end
+    send message, response
+  end
+
+  hear ~r/^dj\s(what|who).*\snext$/i, message do
+    current_playlist = CurrentPlaylistStore.retrieve
+    response =
+      with {:ok, %TlTrack{} = next_track} <- Tracklist.next_track do
+        HedwigMopidy.notice_message("Up next: #{HedwigMopidy.playing_string(next_track, current_playlist)}")
+      else
+        {:error, error_message} -> error_message
+        _ -> HedwigMopidy.notice_message("No more songs are queued")
+      end
+    send message, response
+  end
+
+  hear ~r/^dj\svol(ume)?$/i, message do
     response = with {:ok, level} <- Mixer.get_volume do
       "The volume level is set to #{round(level/10)}"
     end
     send message, response
   end
 
-  hear ~r/^dj\svolume\s(?<level>.*)$/i, message do
+  hear ~r/^dj\svol(ume)?\s(?<level>.*)$/i, message do
```

Comment:
Do we really need to tip the reader that they can either spell out volume or not? It should do the same thing